governance/votes/Executive vote - May 10, 2023.md

```diff
@@ -0,0 +1,156 @@
+* Debt write-off timelock (tau): **0**.
+* Liquidation Process: Will use the established RWA collateral contracts. See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
+
+#### Relevant Smart Contract Information
+
+* **MIP21_LIQUIDATION_ORACLE:** `0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30`. 
+* **RWA_TOKEN_FAB:** `0x2B3a4c18705e99bC29b22222dA7E10b643658552`.
+* **SYMBOL:** `RWA014`.
+* **NAME:** `RWA-014`.
+* **ILK:** `RWA014-A`.
+* **RWA014:** TBD
+* **MCD_JOIN_RWA014_A:** TBD
+* **RWA014_A_URN:** TBD
+* **RWA014_A_JAR:** TBD
+* **RWA014_A_OUTPUT_CONDUIT:** TBD
+* **RWA014_A_INPUT_CONDUIT_URN:** TBD
+* **RWA014_A_INPUT_CONDUIT_JAR:** TBD
+* **RWA014_A_OPERATOR:** TBD
+* **ETH Address A (Coinbase Custody):** TBD
+
+#### Additional Information
+
+Please review the following forum threads containing information about RWA014 (Coinbase Custody) to inform your position before voting:
+* [Initial MIP Forum Thread](https://forum.makerdao.com/t/mip81-coinbase-usdc-institutional-rewards/17703).
+* [CES Domain Team Assessment](https://forum.makerdao.com/t/rwa014-coinbase-custody-interim-ces-domain-team-assessment/19171/1).
+* [Updated Technical Assessment from dewiz](https://forum.makerdao.com/t/rwa014-coinbase-custody-interim-ces-domain-team-assessment/19171/2).
+* [Strategic Finance Core Unit Legal Assessment](https://forum.makerdao.com/t/coinbase-custody-legal-assessment/20384).
+
+### Risk Parameter Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmYFfRuR) the following parameter changes will be made, if this executive proposal passes:
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Changes
+
+* Increase ETH-A Stability Fee by 0.25% from 1.5% to **1.75%**.
+* Increase ETH-B Stability Fee by 0.25% from 3% to **3.25%**.
+* Increase ETH-C Stability Fee by 0.25% from 0.75% to **1%**.
+* Increase WSTETH-A Stability Fee by 0.25% from 1.5% to **1.75%**.
+* Increase WSTETH-B Stability Fee by 0.25% from 0.75% to **1%**.
+* Increase RETH-A Stability Fee by 0.25% from 0.5% to **0.75%**.
+* Increase CRVV1ETHSTETH-A Stability Fee by 0.25% from 1.5% to **1.75%**.
+
+#### [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Changes
+
+* Increase the WSTETH-A `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Increase the WSTETH-B `gap` by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Reduce the WBTC-A `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+* Reduce the WBTC-B `gap` by 5 million DAI from 10 million DAI to **5 million DAI**.
+* Reduce the WBTC-C `gap` by 10 million DAI from 20 million DAI to **10 million DAI**.
+
+#### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Changes
+
+* Increase the WSTETH-A `ttl` by 21,600 seconds from 21,600 seconds to **43,200 seconds**.
```

Comment:
Shouldn't refer to these in seconds. I think it's how it works in the contract, but its unnecessarily hard to parse for governance-facing stuff.

Reply:
added hours in parentheses